packages/gitbook/src/lib/v1.ts

```diff
@@ -134,9 +135,29 @@ async function getDataFetcherV1(): Promise<GitBookDataFetcher> {
 
         getRevision(params) {
             return wrapDataFetcherError(async () => {
-                return getRevision(params.spaceId, params.revisionId, {
+                const { tags, ...revision } = await getRevision(params.spaceId, params.revisionId, {
```

Comment:
It confirms by other comments, you are starting with `computed: false`, so we need to make sure the cache key stays the same as before in this case.